tests/integration_tests/utils.py

```diff
@@ -1071,3 +1072,34 @@ def minio_test_creds():
             }
         }
     }
+
+
+def clear_huggingface_cache():
+    cache_path = os.environ.get("TRANSFORMERS_CACHE")
+
+    if cache_path is None:
+        cache_path = file_utils.default_cache_path.rstrip("/")
+        while not cache_path.endswith("huggingface") and cache_path:
+            cache_path = "/".join(cache_path.split("/")[:-1])
+
+    du = shutil.disk_usage(cache_path)
+
+    logger.info(f"Current disk usage {du} ({100 * du.free / du.total}% usage)")
+
+    # only clean up cache if less than 25% of disk space is used.
+    if du.free / du.total > 0.25:
+        return
+
+    logger.info(
+        f"Clearing HuggingFace cache under path: `{cache_path}`. "
+        f"Free disk space is {100 * du.free / du.total}% of total disk space."
+    )
+    for root, dirs, files in os.walk(cache_path):
+        for f in files:
+            print("trying to clean up", os.path.join(root, f))
```

Comment:
nit: maybe consider using logger instead of prints?

Reply:
will be removing these as they're for debugging